ros2pkg/ros2pkg/resource/ament_python/test_flake8.py.em

```diff
@@ -12,13 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import warnings
```

Comment:
Maybe it's users' resposibility to create `pytest.ini` for their packages if they want to filter the warning.

Reply:
Nice 👍 this is much cleaner and better maintenance. i just stole your idea 🚀 (added you as co-author on this) see https://github.com/ros2/ros2cli/pull/868/commits/e545a5f94bf03e31bda68553833cfc2ddf146c0b